sdk/identity/identity/test/node/managedIdentityCredential.spec.ts

```diff
@@ -167,7 +167,7 @@ describe("ManagedIdentityCredential", function() {
       const bodyParams = qs.parse(authRequest.body);
       assert.equal(authRequest.method, "POST");
       assert.equal(bodyParams.client_id, "client");
-      assert.equal(decodeURIComponent(bodyParams.resource), "https://service");
+      assert.equal(decodeURIComponent(bodyParams.resource as any as string), "https://service");
```

Comment:
Same feedback as https://github.com/Azure/azure-sdk-for-js/pull/8782#discussion_r422329437

We should be able to do just `as string` here